### ISSUE TITLE
[New Feature]: Do not "upgrade" LTS versions of packages to current

### ISSUE THREAD

**Issue by IlgazC** [microsoft/winget-pkgs#89852]
### Description of the new feature/enhancement

When I type in "winget upgrade --all" , it displays this:

```
winget upgrade --all
Name               Id             Version   Available Source
------------------------------------------------------------
MariaDB 10.6 (x64) MariaDB.Server 10.6.11.0 10.9.3.0  winget
```

MariaDB 10.6 is an LTS version that is guaranteed to be maintained for 5 years. Think like Ubuntu LTS Linux LTS. In my situation, it isn't wise/practical to upgrade to the latest version every couple of weeks.

There should be a setting/feature not to upgrade LTS versions to "current" versions however update them to current point releases if it gets a security/stable update. e.g. 10.6.12.0

**Comment by russellbanks** [comment]
There is a feature request for winget about this issue: https://github.com/microsoft/winget-cli/issues/476.

A temporary fix we have been doing is splitting packages into their major versions (for example, `MariaDB.Server.10.6`). Although, *unless these versions can be installed side-by-side*, this is bad practice, and we would have to merge the split packages together again once package pinning is implemented. 

A solution in the meantime is outlined here in https://github.com/microsoft/winget-pkgs/issues/69585#issuecomment-1214181332:
> It's still possible to winget show Microsoft.PowerShell --versions and then winget install Microsoft.PowerShell --version <x.y.z>.

So you can do `winget show MariaDB.Server --versions` and `winget install MariaDB.Server --version <x.y.z>` to get the version you need.

**Comment by stephengillie** [comment]
Yes, creating a `.LTS` (or branch-appropriate suffix) PackageIdentifier is the community-accepted solution for this problem.
